en/docs/install-and-setup/setup/distributed-deployment/synchronizing-artifacts-in-a-gateway-cluster.md

```diff
@@ -30,14 +30,12 @@ The behaviour of the inbuilt Artifact Synchronizer in different scenarios is des
 
 [![artifact synchronizer architecture]({{base_path}}/assets/img/learn/artifact-synchronizer-architecture.png)]({{base_path}}/assets/img/learn/artifact-synchronizer-architecture.png)
 
-1. In the event an API is published, edited, or removed, the Synapse artifact corresponding to the API will be stored/updated in the configured extension point.
+1. If an API is published, edited, or removed, the Synapse artifact corresponding to the API will be stored/updated in the storage through the configured extension point.
 2. An event will be sent to Traffic Manager (TM) using event notifiers with the API Name, UUID, and the Gateway label of the API.
-3. In a distributed deployment, Gateways are subscribed to the Traffic Manager topics. The API Gateway will filter out the events by the Gateway label. It will also sort the APIs that have the Gateway label. 
-4. It will fetch the artifacts associated with the API from the storage (Database or GitHub) and load it to the
- memory.
-
-An extension point needs to be configured in the Publisher profile to store the Synapse artifacts in a persistent storage other than the default API-M database. When an API is published, edited, or removed, an event will be sent to the Traffic Manager using Event Notifiers with the API Name, UUID, and the Gateway label of the API.
-
+3. In a distributed deployment, Gateways are subscribed to the Traffic Manager. The API Gateway will filter out the events by the Gateway label. 
+4. Gateway will request for the artifacts related to the event received from the Traffic manager.
+5. Traffic Manager will connect to the database and fetch the artifacts.
+6. Gateway will receive the artifacts and deploy. If the API is already deployed in the gateway, It will first undeploy it and redeploy with the retrieved artifacts
```

Comment:
```suggestion
6. The Gateway will receive the artifacts and deploy. If the API is already deployed in the Gateway, it will first undeploy it and redeploy with the retrieved artifacts.
```

Reply:
Fixed with c71a601